### PR TITLE
Upgrading libthrift version and recompile thrift code

### PR DESCRIPTION
**Pull request by dilin993**
https://github.com/wso2/product-is/issues/4652

Fix https://github.com/wso2/product-is/issues/4665

### CODE REVIEW

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
-/*
- * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+/**
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (madurangasiriwardena) [inline components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java]: license year should not be changed.

File: components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/thrift/EntitlementService.java [hunk 2]
```diff
@@ -1,2151 +1,2666 @@
 /**
- * Autogenerated by Thrift Compiler (0.7.0)
+ * WSO2 Inc. licenses this file to you under the Apache License,
```
Comment (madurangasiriwardena) [inline components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/thrift/EntitlementService.java]: This is not the correct license header.

File: components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java (hunk 1 above)
Comment (dilin993) [reply components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/ApplicationAuthenticatorService.java]: This file was changed by mistake. I have corrected it.

File: components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/thrift/EntitlementService.java (hunk 2 above)
Comment (dilin993) [reply components/entitlement/org.wso2.carbon.identity.entitlement/src/main/java/org/wso2/carbon/identity/entitlement/thrift/EntitlementService.java]: Updated the licence year as 2016.

**Pull request merged**